productivity-suite/piercing-library/src/stream-utilities.ts

```diff
@@ -21,6 +21,40 @@ export function wrapStreamInText(
   return readable;
 }
 
+export function transformStream(
```

Comment:
I think it's sort of explanatory, but since this is exported (so I guess it's public api) could you add a tsdoc for this function like the one in the function above? 🙏